en/docs/reference/connectors/connectors-overview.md

```diff
@@ -41,65 +41,65 @@ Using connectors provide the following advantages:
     </tr>
     <tr>
         <td><b>150+ available connectors</b></td>
         <td>There are numerous connectors available in the store. This provides multiple options on SAAS providers and popular APIs for you to build your integration use cases as required.</td>
     </tr>
     <tr>
         <td><b>Less domain knowledge required</b></td>
-        <td>When integrating with different APIs, the Connector documentation provides detailed information. Less domain knowledge is required when integrating with systems since all the information about operations are available to you and you do not need to know elaborate details of the system you are integrating with. You can spend less time studying the APIs and focus more on business use cases.</td>
+        <td>When integrating with different APIs, the Connector documentation provides detailed information. Less domain knowledge is required when integrating with systems since all the information about operations is available to you and you do not need to know elaborate details of the system you are integrating with. You can spend less time studying the APIs and focus more on business use cases.</td>
     </tr>
     <tr>
         <td><b>Dynamically added to the runtime</b></td>
         <td>Connectors and most of the extensions can be directly added to and removed from the runtime. It is not required to restart the servers for deployments.</td>
     </tr>
     <tr>
         <td><b>Easy maintenance</b></td>
-        <td>Connectors act as a bridge between main integration logic and the third-party application. Even if the application needs to be updated to support new features, the main integration logic of the API version does not need to be changed.</td>
+        <td>Connectors act as a bridge between the main integration logic and the third-party application. Even if the application needs to be updated to support new features, the main integration logic of the API version does not need to be changed.</td>
     </tr>
 </table>
 
 ## How to use connectors
 
-When configuring the integration logic, you need to use WSO2 Integration Studio. When ready, you can export the project along with dependency connectors to the integration runtime. See [documentation on adding connectors]({{base_path}}/develop/creating-artifacts/adding-connectors/) for more information.
+When configuring the integration logic, you need to use Micro Integrator for VS Code. When ready, you can export the project along with dependency connectors to the integration runtime. See [documentation on adding connectors]({{base_path}}/develop/creating-artifacts/adding-connectors/) for more information.
 
 See the following video for a quick look at how to use connectors.
```

Comment:
Let's remove the video, create an issue to track it, until we come up with a proper video.

Reply:
Updated